UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/IFulfilmentDataService.cs

```diff
@@ -7,5 +7,6 @@ public interface IFulfilmentDataService
     {
         Task<string> CreateExchangeSet(SalesCatalogueServiceResponseQueueMessage message, string currentUtcDate);
         Task<string> CreateLargeExchangeSet(SalesCatalogueServiceResponseQueueMessage message, string currentUtcDate, string largeExchangeSetFolderName);
+        Task GetReadmeFiles(string batchId, string exchangeSetPath, string correlationId);
```

Comment:
No need to add this method in Interface, please remove it.

Reply:
Resolved